spec/01_tic_tac_toe_spec.rb

```diff
@@ -85,40 +103,22 @@
     end
 
     describe '#valid_move?' do
-      it 'returns true/false based on position' do
+      it 'returns true/false based on whether the position is already occupied' do
         game = TicTacToe.new
         board = [" ", " ", " ", " ", "X", " ", " ", " ", " "]
         game.instance_variable_set(:@board, board)
 
-        position = "1"
+        position = 0
         expect(game.valid_move?(position)).to be_truthy
```

Comment:
Can you change the `position` var to `index`? This way we're as clear as possible what we're working with